harness/determined/common/experimental/determined.py

```diff
@@ -64,6 +64,32 @@ def __init__(
 
         self._session = api.Session(master, user, auth, cert)
 
+    def create_user(username: str, password: str, admin: bool):
+        # call API create_user
+        # create User object
+        # return that object
+
+        # new API -> bindings.post_PostUser 
+
+        pass
+
+    def get_user(user_id: int):
+        # return User object
+
+        # new API -> bindings.get_GetUser(user_id) (in api_user.go) ++ need to do username
+        pass
+
+    def get_user(user_name: str):
```

Comment:
python doesn't support overriding like this, you'll either need to accept `Union[int, str]` in `get_user()` or you'll need `get_user_by_id()` and `get_user_by_name()`